plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestingOracleServer.java

```diff
@@ -72,6 +74,8 @@ public class TestingOracleServer
     public TestingOracleServer()
     {
         container = Failsafe.with(CONTAINER_RETRY_POLICY).get(this::createContainer);
+        Logging logging = Logging.initialize();
+        logging.setLevel(RetryingConnectionFactory.class.getName(), DEBUG);
```

Comment:
is this useful? I don't think anyone would see the logs unless they explicitly go looking for it.

Since you've already proved retries work (since it was logged in current CI run) maybe we can remove this.

Let's keep the `log.debug` in the connection factory though - useful to have.